config/foreman-proxy-content.migrations/251105202301-reset-puppet-package-params.rb

```diff
@@ -0,0 +1,13 @@
+if answers["puppet"].is_a?(Hash)
+  puppet = answers["puppet"]
+
+  if puppet.has_key?("server_package") && (puppet["server_package"].empty? ||
+      (puppet["server_package"].is_a?(Array) && puppet["server_package"].empty?))
```

Comment:
That's not what I suggested. Now if `server_package` is `nil`, it will error out as `nil.empty?` doesn't exist.
```suggestion
  if puppet.has_key?("server_package") && (puppet["server_package"].nil? || puppet["server_package"].empty?)
```

Reply:
Ahh I got confused because of the line you commented on, I get you now.

Reply:
Blame the GitHub mobile app.